src/algebra/lie/basic.lean

```diff
@@ -159,14 +159,17 @@ instance : lie_ring_module L (M →ₗ[R] N) :=
       simp only [lie_lie, linear_map.coe_mk, linear_map.map_sub, linear_map.add_apply, lie_sub],
       abel, }, }
 
-@[simp] lemma bracket_apply (f : M →ₗ[R] N) (x : L) (m : M) : ⁅x, f⁆ m = ⁅x, f m⁆ - f ⁅x, m⁆ := rfl
+@[simp] lemma lie_hom.bracket_apply (f : M →ₗ[R] N) (x : L) (m : M) :
```

Comment:
```suggestion
@[simp] lemma lie_hom.lie_apply (f : M →ₗ[R] N) (x : L) (m : M) :
```
But may this would require more changes in other places as well?

Reply:
Better, thanks.